diagnostic.md

```diff
@@ -6,72 +6,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The puropse of a backend is to save data. It allows data to persist when browser is closed.
+In addition, it allows two users to interact with each other in the app
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+model fetches data in the database
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+controller(lazy- most often skinny- manager) communicatese with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+We do not need views . We are using our own view layer. My Question(when there it is a single page app, is the view neccessary?).
 ```
```

Comment:
Nope, we don't use the rails view layer for single page apps.  If you're creating a multi-page app or using some sort of templating you may use the rails view layer.